client-desktop/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "raven-client-desktop",
+  "productName": "Raven Notes Test2",
```

Comment:
Raven Notes. Get out from the test zone.

Reply:
Competely forgot about that line